Content/default/tests/Client/Client.Tests.fs

```diff
@@ -5,27 +5,29 @@ open Fable.Mocha
 open Index
 open Shared
 
-let client = testList "Client" [
-    testCase "Added todo" <| fun _ ->
-        let newTodo = Todo.create "new todo"
-        let model, _ = init ()
+let client =
+    testList
+        "Client"
+        [ testCase "Added todo"
+          <| fun _ ->
+              let newTodo = Todo.create "new todo"
+              let model, _ = init ()
 
-        let model, _ = update (AddedTodo newTodo) model
+              let model, _ = update (AddedTodo newTodo) model
 
-        Expect.equal 1 model.Todos.Length "There should be 1 todo"
-        Expect.equal newTodo model.Todos.[0] "Todo should equal new todo"
-]
+              Expect.equal 1 model.Todos.Length "There should be 1 todo"
+              Expect.equal newTodo model.Todos.[0] "Todo should equal new todo" ]
 
 let all =
-    testList "All"
+    testList
+        "All"
         [
-//-:cnd:noEmit
+          //-:cnd:noEmit
```

Comment:
not sure if this extra space here won't fool the template engine

Reply:
Hmm, perhaps to be on the safe side (pun intended), the test files [should be ignored](https://github.com/fsprojects/fantomas/blob/master/docs/Documentation.md#ignore-files-fantomasignore).

Reply:
Just tested it - indeed the engine doesn't like it, this is what gets produced:
```fsharp
let all =
    testList
        "All"
        [
          #if FABLE_COMPILER // This preprocessor directive makes editor happy
          Shared.Tests.shared
#endif
                    client ]
```

Yeah I'm thinking maybe start small by formatting only `src/` files and leave `tests/` for now?

Reply:
Good idea!